scripts/common/common.sh

```diff
@@ -327,11 +327,9 @@ exportKubeconfig() {
     local kubeconfig
     kubeconfig="$(${K8S_DISTRO}_get_kubeconfig)"
 
-    current_user_sudo_group
-    if [ -n "$FOUND_SUDO_GROUP" ]; then
-        chown root:$FOUND_SUDO_GROUP ${kubeconfig}
-    fi
-    chmod 440 ${kubeconfig}
+    chown root:root ${kubeconfig}
+    chmod 444 ${kubeconfig}
```

Comment:
I just know someone's going to complain to us about this, and I think a much better long term solution would be to make a copy of the kubeconfig file that's owned by the sudo user group

but this'll work for the moment

Reply:
I think we should put this behind the feature flag that Lavery added in the spec.

Reply:
I updated this file now to check for the CIS_COMPLIANCE flag that Andrew introduced